app/_locales/en/messages.json

```diff
@@ -1,14 +1,14 @@
 {
   "appName": {
-    "message": "Behave!",
+    "message": "Behave! — Offensive Comments Blocker",
```

Comment:
But "Blocker" is a simple wording. Less geeky. And I think Alex has done some research on it for better store visibility, so I am OK with it.